Makefile

```diff
@@ -1,8 +1,8 @@
 CC		= g++
-CFLAGS	= -O2 -Wall
+CFLAGS	= -std=c++14 -O2 -Wall
```

Comment:
Why should I add -std = c + + 14? I'd like to be able to compile with C + + 11 as much as possible, but where is the problem?

Reply:
Because the build fails if I use C++ 11. The error message is below:

```
src/VCFHeteroHomo.cpp: In member function ‘std::pair<std::vector<VCFHeteroHomo*>, std::vector<VCFHeteroHomoRecord*> > VCFHeteroHomo::determine_haplotype(const OptionImpute*) const’:
src/VCFHeteroHomo.cpp:209:98: error: use of ‘auto’ in lambda parameter declaration only available with ‘-std=c++14’ or ‘-std=gnu++14’
  209 |                                                                                         [](const auto& a, const auto& b) {
      |                                                                                                  ^~~~
src/VCFHeteroHomo.cpp:209:113: error: use of ‘auto’ in lambda parameter declaration only available with ‘-std=c++14’ or ‘-std=gnu++14’
  209 |                                                                                         [](const auto& a, const auto& b) {
      |                                                                                                                 ^~~~
src/VCFHeteroHomo.cpp: In lambda function:
src/VCFHeteroHomo.cpp:210:106: error: request for member ‘size’ in ‘a’, which is of non-class type ‘const int’
  210 |                                                                                                 return a.size() < b.size(); });
      |                                                                                                          ^~~~
src/VCFHeteroHomo.cpp:210:117: error: request for member ‘size’ in ‘b’, which is of non-class type ‘const int’
  210 |                                                                                                 return a.size() < b.size(); });
      |                                                                                                                     ^~~~
In file included from /usr/include/c++/11/bits/stl_algobase.h:71,
                 from /usr/include/c++/11/deque:60,
                 from /usr/include/c++/11/stack:60,
                 from src/VCFHeteroHomo.cpp:1:
```

The full log can be found here for your reference.

https://github.com/jcpineda15/Graphite/actions/runs/6636044700/job/18027923140

Reply:
I will correct them.

Reply:
I've updated.